flask_ninja/models.py

```diff
@@ -0,0 +1,385 @@
+    maxItems: Optional[int] = Field(default=None, gte=0)
+    minItems: Optional[int] = Field(default=None, gte=0)
+    uniqueItems: Optional[bool] = None
+    maxProperties: Optional[int] = Field(default=None, gte=0)
+    minProperties: Optional[int] = Field(default=None, gte=0)
+    required: Optional[List[str]] = None
+    enum: Optional[List[Any]] = None
+    type: Optional[str] = None
+    allOf: Optional[List["Schema"]] = None
+    oneOf: Optional[List["Schema"]] = None
+    anyOf: Optional[List["Schema"]] = None
+    not_: Optional["Schema"] = Field(default=None, alias="not")
+    items: Optional[Union["Schema", List["Schema"]]] = None
+    properties: Optional[Dict[str, "Schema"]] = None
+    additionalProperties: Optional[Union["Schema", Reference, bool]] = None
+    description: Optional[str] = None
+    format: Optional[str] = None
+    default: Optional[Any] = None
+    nullable: Optional[bool] = None
+    discriminator: Optional[Discriminator] = None
+    readOnly: Optional[bool] = None
+    writeOnly: Optional[bool] = None
+    xml: Optional[XML] = None
+    externalDocs: Optional[ExternalDocumentation] = None
+    example: Optional[Any] = None
+    deprecated: Optional[bool] = None
+
+    class Config:
+        extra: str = "allow"
+
+
+class Example(BaseModel):
+    summary: Optional[str] = None
+    description: Optional[str] = None
+    value: Optional[Any] = None
+    externalValue: Optional[AnyUrl] = None
+
+    class Config:
+        extra = "allow"
+
+
+class ParameterInType(Enum):
+    query = "query"
+    header = "header"
+    path = "path"
+    cookie = "cookie"
+
+
+class Encoding(BaseModel):
+    contentType: Optional[str] = None
+    headers: Optional[Dict[str, Union["Header", Reference]]] = None
+    style: Optional[str] = None
+    explode: Optional[bool] = None
+    allowReserved: Optional[bool] = None
+
+    class Config:
+        extra = "allow"
+
+
+class MediaType(BaseModel):
+    schema_: Optional[Union[Schema, Reference]] = Field(default=None, alias="schema")
+    example: Optional[Any] = None
+    examples: Optional[Dict[str, Union[Example, Reference]]] = None
+    encoding: Optional[Dict[str, Encoding]] = None
+
+    class Config:
+        extra = "allow"
+
+
+class ParameterBase(BaseModel):
+    description: Optional[str] = None
+    required: Optional[bool] = None
+    deprecated: Optional[bool] = None
+    # Serialization rules for simple scenarios
+    style: Optional[str] = None
+    explode: Optional[bool] = None
+    allowReserved: Optional[bool] = None
+    schema_: Optional[Union[Schema, Reference]] = Field(default=None, alias="schema")
+    example: Optional[Any] = None
+    examples: Optional[Dict[str, Union[Example, Reference]]] = None
+    # Serialization rules for more complex scenarios
+    content: Optional[Dict[str, MediaType]] = None
+
+    class Config:
+        extra = "allow"
+
+
+class Parameter(ParameterBase):
+    name: str
+    in_: ParameterInType = Field(alias="in")
```

Comment:
you can try using `allow_population_by_field_name` to allow creation of model using `in_` field name

```python
class Parameter(ParameterBase):
    name: str
    in_: ParameterInType = Field(alias="in")
    
    class Config:
        allow_population_by_field_name = True    
```